jcl/src/java.base/share/classes/openj9/internal/criu/security/CRIUConfigurator.java

```diff
@@ -48,7 +47,7 @@ public final class CRIUConfigurator {
 	private static final HashMap<String, String> oldProviders = new HashMap<>();
 	/** Tracing for CRIUSEC. */
 	private static final boolean debug = Boolean.parseBoolean(
-			GetPropertyAction.privilegedGetProperty("enable.j9internal.checkpoint.security.api.debug", "false"));
+			System.getProperty("enable.j9internal.checkpoint.security.api.debug", "false"));
```

Comment:
This must continue to use `GetPropertyAction` for older versions; suggest:
```java
	private static final boolean debug = Boolean.parseBoolean(
			/*[IF JAVA_SPEC_VERSION < 24]*/
			sun.security.action.GetPropertyAction.privilegedGetProperty
			/*[ELSE] JAVA_SPEC_VERSION < 24
			System.getProperty
			/*[ENDIF] JAVA_SPEC_VERSION < 24 */
					("enable.j9internal.checkpoint.security.api.debug", "false"));
```

Reply:
Ah, fixed.